usagov/vote-gov-drupal
# Feature/vote 999/migrate meta data

## Jira ticket (required)
https://bixal-projects.atlassian.net/browse/VOTE-999

## Description (optional)
Migrated the default meta tags for all languages on the current site.
-Global
-Front page
-Content: Landing page, state page

## Deployment and testing (required, if applicable)

### Post-deploy steps
1. lando retune

### Testing steps
1. For each page type, view page source in the browser
2. Confirm that all meta tags defined in the ticket match with the current site
